test/parallel/test-net-remote-address-port.js

```diff
@@ -34,11 +34,13 @@ const remoteFamilyCandidates = ['IPv4'];
 if (common.hasIPv6) remoteFamilyCandidates.push('IPv6');
 
 const server = net.createServer(common.mustCall(function(socket) {
-  // Test to see real value in CI log
-  assert.match(socket.remoteAddress,
-               /^127\.0\.0\.1$|^::1$|^::ffff:127\.0\.0\.1$/);
-  // REM: assert.ok(remoteAddrCandidates.includes(socket.remoteAddress));
-  assert.ok(remoteFamilyCandidates.includes(socket.remoteFamily));
+  // REM: assert.match(socket.remoteAddress,
+  // REM:              /^127\.0\.0\.1$|^::1$|^::ffff:127\.0\.0\.1$/);
+  assert.ok(remoteAddrCandidates.includes(socket.remoteAddress),
+            `Illformed remoteAddress: ${socket.remoteAddress}`);
```

Comment:
I know it was my suggestion, but we should use `invalid` instead of `illformed`, since we're not validating the form but the content.

Reply:
```suggestion
            `Invalid remoteAddress: ${socket.remoteAddress}`);
```